jenkins-admin/managed-config-files/custom/clouds-yaml/customizedCredentialMappings.sandbox.yaml

```diff
@@ -0,0 +1,4 @@
+---
+customizedCredentialMappings:
+  - credentialsId: "dummy"
+    tokenKey: "dummykey"
```

Comment:
The tokenKey should be "CLOUD" sinde in the content file you are calling "CLOUD_USR" and "CLOUD_PSW", the token name in this case is "CLOUD"

Reply:
sure, noted that part,was bootstrapping the files. Will create them